### PR TITLE
docs(webpack): add webpack3 boilerplate with theming

### PR DESCRIPTION
**Pull request by layershifter**
Rel #1636.

This PR adds the missed information about theming and adds the real boilerplate 😺 

## Boilerplate features

- preconfigured theming
- CSS modules for user components
- hot module reload
- optimized build (automatic direct imports, proptypes removal)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,11 +75,8 @@ These great products are built on Semantic UI React. Add yours [here][22].
 
 This is a listing of example projects and guides that will help you integrate Semantic UI React into your new or existing projects.
 
-### [webpack1][28]
-Our example project right [here][28] in this repo.
- 
-### [webpack2][29]
-Our example project right [here][29] in this repo.
+### [webpack][28]
+Our example project right [here][28] in this repo. Includes theming examples.
```
Comment (layershifter) [inline README.md:79]: Webpack 1&2 is outdated, we need only one boilerplate for latest version.

File: README.md [hunk 2]
```diff
@@ -199,12 +196,11 @@ Big thanks to our [contributors][20], especially:
 [21]: https://github.com/Semantic-Org/Semantic-UI-React/labels/good%20first%20issue
 [22]: https://github.com/Semantic-Org/Semantic-UI-React/edit/master/README.md
 [23]: https://react.semantic-ui.com/usage#css
-[24]: https://github.com/Semantic-Org/Semantic-UI-React/issues/802#issuecomment-258990274
+[24]: https://medium.com/webmonkeys/webpack-2-semantic-ui-theming-a216ddf60daf
```
Comment (layershifter) [inline README.md:199]: I've updated link as information there has more value for our users.

**Comment by layershifter** [comment]
@brianespinosa @levithomason It will be awesome to get feedback from you there. Grammar fixes is much appreciated 😸

**Comment by levithomason** [comment]
Very cool, thanks for this!  I'm just wrapping my session for today.  I want to give this PR the attention it deserves, so that may come at a later date. ❤️

File: docs/app/Views/Theming.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        content='Configuring Webpack'
+        labelPosition='right'
+        href='https://medium.com/webmonkeys/webpack-2-semantic-ui-theming-a216ddf60daf'
+        icon='medium'
+        target='_blank'
+      />
+
+      <Header as='h2' content='Boilerplate' dividing />
+      <p>
+        If you are looking a boilerplate, we prepared it. It powered by latest Webpack and includes the all required
```
Comment (brianespinosa) [inline docs/app/Views/Theming.js]: "The boilerplate we prepared is powered by the latest Webpack and includes the all required things for quick start like preconfigured theming and hot module reload."

**Review by brianespinosa — COMMENTED** [review]
Everything else looks good. Just a few grammar and copy changes I am suggesting.

I saw that we are also using the beta version of babel-loader and @babel. Is this for JSX fragments? I try to stay away from beta releases on packages if I can avoid it.

**Comment by layershifter** [comment]
> I saw that we are also using the beta version of babel-loader and @babel. Is this for JSX fragments? I try to stay away from beta releases on packages if I can avoid it.

Babel 7 is much faster and enough stable now. I'm use it in my project last few weeks and I really like it.

I almost sure that it will be released soon as stable and it makes no sence to make bet on Babel 6 now to me.

**Review by brianespinosa — APPROVED** [review]

**Review by pratikt-cuelogic — APPROVED** [review]

**Comment by levithomason** [comment]
Looks like we have approvals here. In the future @layershifter, if you get multiple approvals and are confident, please feel free to merge without me. I don't want to hinder progress and I have trust for everyone we're adding to the team.

/cc @brianespinosa @fracmak

Note, @layershifter has been here a long time and has master merge rights. I control releases, but anything that makes it to master will make the next release cycle.

**Comment by levithomason** [comment]
Looks a bad merge on this guy. Build is failing. Might need to roll it back, or make a fix PR. See master on CircleCI.

**Comment by layershifter** [comment]
It's a long term problem with tests:

> FAILED TESTS:
  ✖ "after each" hook
    HeadlessChrome 0.0.0 (Linux 0.0.0)
    Error: Uncaught RangeError: Maximum call stack size exceeded (src/lib/index.js?504b:178:0

I can't to find out the reason locally, very strange issue. Last build was successed.

**Comment by levithomason** [comment]
Random intermittent failure I suppose.  Thanks.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.78.3`.

**Pull request merged**